tests/test_diffraction_objects.py

```diff
@@ -191,7 +191,7 @@ def test_init_invalid_xtype():
     "org_do_args, target_do_args, scale_inputs, expected",
     [
         # Test whether scale_to() scales to the expected values
```

Comment:
`scales_to() scales to` feels a bit circulating?

Reply:
Oops yes forgot to change this - thanks!